apis/sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/web/resources/GraphqlCache.java

```diff
@@ -145,11 +146,12 @@ CompletionStage<Optional<GraphQL>> getGraphql(StargateBridgeClient bridge) {
       CompletionStage<Optional<Schema.CqlKeyspaceDescribe>> keyspaceFuture =
           bridge.getKeyspaceAsync(keyspaceName, true);
 
-      return keyspaceFuture.thenCompose(
+      return keyspaceFuture.thenComposeAsync(
           maybeKeyspace ->
               maybeKeyspace
                   .map(keyspace -> handleExisting(keyspace, bridge))
-                  .orElse(handleMissing()));
+                  .orElse(handleMissing()),
+          Infrastructure.getDefaultWorkerPool());
```

Comment:
1. Not super happy about that. I'm aware that the correct solution would be to rewrite everything from scratch to use async/reactive bridge methods only, but from a practical standpoint I'm not sure if we want to invest this effort right now.
2. I'm not sure if this is the best executor to use, I haven't found good docs about Quarkus' threading model.

Reply:
Ticket for async: #421